### PR TITLE
show more details from reviews in a collapsible

### PR DESCRIPTION
**Pull request by columbiadining**

### CODE REVIEW

File: public/css/skeleton.css [hunk 1]
```diff
@@ -7,7 +7,663 @@
...
 */
 
+/*
+* Skeleton V2.0.4
+* Copyright 2014, Dave Gamache
+* www.getskeleton.com
+* Free to use under the MIT license.
+* http://www.opensource.org/licenses/mit-license.php
+* 12/29/2014
+*/
```
Comment (niharikasingh) [inline public/css/skeleton.css]: This section seems like a duplicate of the already existing CSS styles. Is that right?

File: public/js/evaluations.js [hunk 2]
```diff
@@ -437,16 +437,67 @@ function displayDopeEvals(data) {
...
+    if (data[i].comments && data[i].comments.length != 0) {
+      appendStr += "<li class='dopeComment' tabindex='0'>";
+      appendStr += data[i].comments + "</li>";
+    }
+  }
+  appendStr += "</ul>"
+
+  // display details of each submission in a collapsible
+  // TODO: we might also want to have a field for course and professor, for profs who teach more than one course or vice versa.
+  appendStr += '<button type="button" class="collapsible">See Detailed Reviews</button>'
```
Comment (niharikasingh) [inline public/js/evaluations.js]: What do you think about rewording this to "See Individual Reviews"?

File: public/js/evaluations.js [hunk 3]
```diff
@@ -437,16 +437,67 @@ function displayDopeEvals(data) {
...
+  for (i = 0; i < data.length; i++) {
+    if (data[i].comments && data[i].comments.length != 0) {
+      appendStr += "<li class='dopeComment' tabindex='0'>";
+      appendStr += data[i].comments + "</li>";
+    }
+  }
+  appendStr += "</ul>"
+
+  // display details of each submission in a collapsible
+  // TODO: we might also want to have a field for course and professor, for profs who teach more than one course or vice versa.
```
Comment (niharikasingh) [inline public/js/evaluations.js]: I think this new code for assembling+showing the individual reviews should go into its own function, maybe something like `displayIndividualReviews`. That will keep the functions for the different parts of the display separate. 

You'll also need to create a new div on the evaluations page that holds this, and then use that in a line that corresponds to `$("#displayDopeComments").append(appendStr + "</ul></div>");`

File: server/evaluations.js [hunk 4]
```diff
@@ -33,11 +33,11 @@ exports.setApp = function (app, pool) {
     course = course.replace(/[^-a-z0-9 \/]/g , "");
     var professor = searchParams["professor"]
     professor = professor.replace(/[^-a-z0-9 \/]/g , "");
-    var queryString = "SELECT comments FROM dopeevals WHERE";
+    var queryString = "SELECT * FROM dopeevals WHERE";
     if (course.length == 0) queryString += util.format(" professor LIKE '%%%s'", professor);
     else if (professor.length == 0) queryString += util.format(" course LIKE '%%%s%%'", course);
     else queryString += util.format(" professor LIKE '%%%s' AND course LIKE '%%%s%%'", professor, course);
-    queryString += " AND comments IS NOT NULL ORDER BY adddate DESC;";
+    queryString += "AND GRADES <> 0 ORDER BY adddate DESC;";
```
Comment (niharikasingh) [inline server/evaluations.js]: What's the reasoning for removing the leading space? I thought that would be needed to construct the query, to make sure it doesn't look like `...course LIKE 'environmental'AND GRADES...`. Also, I think grades should be lowercase for style consistency.

**Review by niharikasingh — CHANGES_REQUESTED** [review]

**Comment by niharikasingh** [comment]
Thanks for doing this! I've made a few comments - let me know if anything doesn't make sense. I really appreciate your help with this project. 

Could you please add a screenshot of the new feature to the top comment in this PR? That will be good documentation for later.

File: public/css/skeleton.css (hunk 1 above)
Comment (columbiadining) [reply public/css/skeleton.css]: Most of these are formatting changes suggested by my linter!
I added some lines but other than that, no code changes.
The additions I made were lines 583 - 604:

```suggestion
*/
/* Style the button that is used to open and close the collapsible content */

.collapsible {
	background-color: #eee;
	color: #444;
	cursor: pointer;
	padding: 18px;
  width: 100%;
  height: 70px;
	border: none;
	text-align: left;
	outline: none;
	font-size: 15px;
}


/* Add a background color to the button if it is clicked on (add the .active class with JS), and when you move the mouse over it (hover) */

.active,
.collapsible:hover {
	background-color: #ccc;
}


/* Style the collapsible content. Note: hidden by default */

.content {
	padding: 0 18px;
	display: none;
	overflow: hidden;
	background-color: #f1f1f1;
}

```

File: public/js/evaluations.js (hunk 2 above)
Comment (columbiadining) [reply public/js/evaluations.js]: That sounds good, will revise to do this.

File: server/evaluations.js (hunk 4 above)
Comment (columbiadining) [reply server/evaluations.js]: Thanks for the catch! I added the grades filter last-minute to filter out dummy reviews, so must have missed the inconsistency when I published this. Fixed in next revision.

File: public/js/evaluations.js (hunk 3 above)
Comment (columbiadining) [reply public/js/evaluations.js]: Done in next revision!

**Pull request closed without merging**